jcl/src/java.base/share/classes/java/lang/System.java

```diff
@@ -473,7 +478,11 @@ static void completeInitialization() {
 	/*[ENDIF]*/ // Sidecar18-SE-OpenJ9
 
 	/*[IF (Sidecar18-SE-OpenJ9|Sidecar19-SE)&!(PLATFORM-mz31|PLATFORM-mz64)]*/
-	setIn(new BufferedInputStream(new FileInputStream(FileDescriptor.in)));
+	InputStream tempIn = new BufferedInputStream(new FileInputStream(FileDescriptor.in));
+	setIn(tempIn);
+/*[IF JAVA_SPEC_VERSION >= 20]*/
+	initialIn = tempIn;
```

Comment:
This is redundant: the assignment to `initialIn` has already happened within `setIn()`.
Did you intend to remove the addition to `setIn()`?

Reply:
FYI: @JasonFengJ9

Reply:
https://github.com/eclipse-openj9/openj9/blob/82d8e4396b1c29eb28bf38adec66cda4ec1b34a5/jcl/src/java.base/share/classes/java/lang/System.java#L547
`setIn()` is a public API, the `in` can be modified by the application.
`Access.initialSystemIn()` returns the first one set by bootstrap code.

Reply:
@keithc-ca the assignment is not done in setIn() (any more), are you looking at some old code?

Reply:
Perhaps I was.